src/gflip/gflip_engine.cpp

```diff
@@ -358,10 +443,134 @@ void gflip_engine::run_evaluation(int dtype)
 
 // ---------------------------------------------------------
 
+void gflip_engine::update_tfidf()
+{
+  {
+    ScopedTimer scopedTimer("Adding docs");
+    for (int i=0; i<laserscan_bow.size(); ++i)
+    {
+      add_doc_stats(i);
+    }
+  }
+
+  {
+    ScopedTimer scopedTimer("Updating tf_idf");
+    compute_idfs();
+    compute_tf_idfs();
+  }
+
+  {
+    ScopedTimer scopedTimer(" tf_idf");
+    for(uint i=0;i<laserscan_bow.size();i++)
+      laserscan_bow[i].norm_wgv = norm_gfp(laserscan_bow[i]);
+  }
+
+}
+
+// ---------------------------------------------------------
+
+
+void gflip_engine::update_tfidf(int nscans)
```

Comment:
Nope this is basically for the last nscans that were inserted into laserscan_bow.size but are not processed into the tf_idf yet